kofta/src/webrtc/components/AudioRender.tsx

```diff
@@ -74,8 +85,7 @@ export const AudioRender: React.FC<AudioRenderProps> = () => {
       >
         <div className={`p-8 rounded m-auto bg-simple-gray-3c`}>
           <div className={`text-center mb-4`}>
-            Browsers require user interaction before they will play audio. Just
-            click okay to continue.
+            Are you sure you want to join {room?.name}?
```

Comment:
I feel this might make some users think it's a hassle to click join two times. Whereas the previous one clearly stated it's their browsers fault and they need to interact with it once.

Reply:
Also i think adding this to the translations would be a good idea too.

Reply:
I approved by mistake so this comment is for requesting changes

Reply:
This is how most platforms like Google Meet, Zoom etc solve the problem of requiring interactions. Browser Interaction message is not as user friendly

Reply:
@ArnavBansal11 that was just my opinion. If other people approve even ill approve but still, add the translation keys. I think this is a important thing that needs to be translated

Reply:
Ok i will add translation keys, but I will have to keep them in English for now due to my limited knowledge in linguistics

Reply:
Yea sure!

Reply:
@amitojsingh366 this would only happen when a user joins through a link no?

Reply:
yep this will only happen then, not if a user joins through the list

Reply:
@jamesql yes that's correct, or might even happen if they're in a room and then open the site from another device *(throws you back into the room)*